blog/fine-grained-news-2024-07.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Fine Grained News - July 2024
+description: Fine Grained News
+slug: fine-grained-news-2024-07
+date: 2024-07-31
+authors: hello-caleb
+tags: [newsletter]
+image: https://openfga.dev/img/og-rich-embed.png
+hide_table_of_contents: false
+---
+# Fine Grained News
+
+
+Welcome to the July 2024 edition of Fine Grained News! We are thrilled to bring you the latest updates, features, and community highlights from OpenFGA. This month has included releases, performance improvements, and insights shared through our community meetings and presentations.
+
+
+We value your feedback and invite you to participate in our [2024 OpenFGA Community Survey](https://www.surveymonkey.com/r/OPENFGA2024). Your insights help us understand your needs better and improve our offerings. Please take a few minutes to complete the survey and let your voice be heard.
+
+
+## Improvements
+
+
+### Latest Features
+
+
+- We’ve introduced consistency options for query requests. This new feature provides more flexibility and control over how queries are executed, enhancing the accuracy and reliability of query results. [Learn more about this update](https://openfga.dev/blog/stronger-consistency-options-announcement).
+
+
+- We’re now publishing images to `ghcr.io/openfga/openfga` as an alternative to DockerHub, thanks to the contribution from @JAORMX. This provides an additional option for accessing and deploying our containers. [Read more](https://github.com/openfga/openfga/commit/#1775).
+
+
+### Performance Improvements
+
+
+- We have implemented significant performance enhancements in the Check function, ensuring that computed relations do not consume from the resolution depth quota, do not trigger additional goroutines, and do not get cached. These optimizations lead to more efficient access control checks and improved overall system performance. [Learn more](https://github.com/openfga/openfga/commit/#1786).
+
+
+- We have made significant performance improvements to userset subproblem resolutions in Check in certain scenarios. These enhancements optimize the efficiency of access control checks, leading to faster processing times and improved system performance. [Read more about this improvement](https://github.com/openfga/openfga/commit/#1734).
+
+
+- We have also improved the performance of tuple-to-userset subproblem resolutions in Check for certain scenarios. This update enhances the speed and efficiency of complex access control queries, further boosting overall system performance. [Learn more about this update](https://github.com/openfga/openfga/commit/#1735).
```

Comment:
Could we collapse all these into one bullet please?

- We have implemented significant performance enhancements in the Check function. See https://github.com/openfga/openfga/commit/#1786, https://github.com/openfga/openfga/commit/#1734 and https://github.com/openfga/openfga/commit/#1735

I don't think people will know what `computed relations do not consume from the resolution depth quota`, `userset subproblem resolutions`, `tuple-to-userset subproblem resolutions` even mean 🤷‍♀️

Reply:
Yes, I agree

Reply:
Sorry, those links don't work properly. Please use

- https://github.com/openfga/openfga/pull/1786
- https://github.com/openfga/openfga/pull/1734
- https://github.com/openfga/openfga/pull/1735

Reply:
Updated to: We've improved our Check latency up to 20X in some scenarios in 1.5.7 and 1.5.6.